simplejson/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```

Comment:
Same comment as abouve about the `--single-version-externally-managed --record record.txt`